uint32 id
float64 r
float64[] pose0
dwal_planner/Path[] paths
